src/remote/dto/bio.auth.save.params.dto.ts

```diff
@@ -0,0 +1,15 @@
+import { FingerprintTypeEnum } from '../fingerprint.type.enum';
+import { IsEnum, IsInt, IsISO8601, IsNumber, IsOptional, IsString, Length } from 'class-validator';
+
+/**
+ * DTO for the params of each fingerprint in the body of the request sent to Bio Auth Service's /save endpoint
+ */
+export class BioAuthSaveParamsDto {
+    @IsInt() readonly type_id: number;
```

Comment:
What about position?

Reply:
Ack, thank you, good catch.

Reply:
Fixed in the latest and tested against protocol's integration tests.